EffectsTest/Game.cpp

```diff
@@ -927,27 +942,27 @@ void Game::CreateDeviceDependentResources()
         m_alphaTestNotEqual = std::make_unique<AlphaTestEffect>(device, EffectFlags::None, pdOpaque, D3D12_COMPARISON_FUNC_NOT_EQUAL);
 
         //--- NormalMapEffect --------------------------------------------------------------
-        m_normalMapEffect = std::make_unique<NormalMapEffect>(device, EffectFlags::None, pdOpaque);
+        m_normalMapEffect = std::make_unique<NormalMapEffect>(device, EffectFlags::Specular, pdOpaque);
```

Comment:
The default parameter was ``bool specular = true`` so it needs to be ``EffectFlags::Specular`` to match that.